src/MasterSite.php

```diff
@@ -1248,16 +1248,27 @@ public function add_image_attachment_fields_to_save( $post, $attachment ) {
 	}
 
 	/**
-	 * Override the Gutenberg core/image block render method output, To add credit field in it's caption text.
+	 * Override the Gutenberg core/image block render method output, To add credit field in it's caption text & image alt text as title.
 	 *
 	 * @param array  $attributes    Attributes of the Gutenberg core/image block.
 	 * @param string $content The image element HTML.
 	 *
-	 * @return string HTML content of image element with credit field in caption.
+	 * @return string HTML content of image element with credit field in caption and alt text in image title.
 	 */
 	public function p4_core_image_block_render( $attributes, $content ) {
-		$image_id = isset( $attributes['id'] ) ? trim( str_replace( 'attachment_', '', $attributes['id'] ) ) : '';
-		$credit   = $image_id ? get_post_meta( $image_id, self::CREDIT_META_FIELD, true ) : '';
+		$image_id      = isset( $attributes['id'] ) ? trim( str_replace( 'attachment_', '', $attributes['id'] ) ) : '';
+		$img_post_meta = $image_id ? get_post_meta( $image_id ) : [];
+		if ( ! $img_post_meta ) {
+			return $content;
```

Comment:
Sonarcloud complains about this 4th return statement that's added (it only allows 3). Though in this case the added return does make sense, so not sure how we should please Sonar :stuck_out_tongue:

Reply:
Removed the one return statement, hope Sonarcloud is happy now 😄